example/25-rbac.yaml

```diff
@@ -0,0 +1,79 @@
+# SPDX-FileCopyrightText: 2021 SAP SE or an SAP affiliate company and Gardener contributors
```

Comment:
I am not sure if RBACs are needed here, but
- the permissions here differ from the one in the charts
- there is no binding, so no user is actually using them at the moment.